src/content/contributing/translation-program/mission-and-vision/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Mission and vision
+lang: en
+description: The ethereum.org Translation Program mission and vision
+sidebar: true
+---
+
+# Mission and vision {#mission-and-vision}
```

Comment:
<img width="1071" alt="Screenshot 2022-08-12 at 11 51 40" src="https://user-images.githubusercontent.com/62268199/184339930-4b4bf376-138d-49c4-9844-2689b9b6376f.png">

Has no table of content items because no headers